lib/install.js

```diff
@@ -444,3 +407,60 @@ function install (gyp, argv, callback) {
   }
 
 }
+
+function download (gyp, env, url, cb) {
```

Comment:
taking a cb option that's only used when erroring is a bit gross, perhaps it'd closer to idiomatic if the thrown error bubbled up and the try/catch was done at the `download()` call?